zips/draft-jackgavigan-defer-and-extend.rst

```diff
@@ -0,0 +1,186 @@
+
+The character § is used when referring to sections of the Zcash Protocol
+Specification. [#protocol]_
+
+The term "Mainnet" is to be interpreted as described in § 3.12 ‘Mainnet and 
+Testnet’. [#protocol-networks]_
+
+"Lockbox" refers to the in-protocol lockbox described in ZIP 1015, [#zip-1015]_ 
+and specified in ZIP 2001. [#zip-2001]_
+
+"Current Dev Fund" refers to the block subsidy allocation described in ZIP 1015, 
+[#zip-1015]_ and specified in ZIP 2001. [#zip-2001]_
+
+"Zcash Community Grants", also called "ZCG", refers to the committee described 
+in [#zip-1015]_.
+
+"Bootstrap Project", also called "BP", refers to the 501(c)(3) nonprofit 
+corporation of that name.
+
+"Zcash Foundation", also called "ZF", refers to the 501(c)(3) nonprofit 
+corporation of that name.
+
+"Electric Coin Company", also called "ECC", refers to the Zerocoin Electric 
+Coin Company, LLC.
+
+
+Abstract
+========
+This ZIP proposes expressly prohibiting any disbursement of funds from the 
+Lockbox until after Zcash Shielded Assets is active on Mainnet, and extending 
+the Current Dev Fund for one year. 
+
+
+Motivation
+==========
+
+Zcash Shielded Assets (ZSAs) is arguably the single most impactful change to the 
+Zcash protocol since the launch of Zcash in late 2016. There is clear consensus
+support for ZSAs across the Zcash community, and a ZCAP poll carried out in 
+August 2021 indicated that the Zcash community believed that ZSAs should be 
+first priority for Zcash following the activation of NU5. [#priorities]_
+
+In January 2022, ZCG provided funding for QEDIT to research and implement ZSAs 
+[#zsagrant]_, and the functionality was demonstrated using a fork of zcashd 
+during Zcon4 in August 2023. [#zsademo]
+
+Due to ECC's decision to no longer support new protocol features in the zcashd 
+node implementation, zcashd must be deprecated before ZSAs can activate on 
+Mainnet. Zcashd deprecation was expected to be complete by April 2025. However, 
+the necessary work was not prioritized. [#nu7-timeline]_ As a result, as of March 
+2025, there is no confirmed schedule for zcashd deprecation (and, therefore, no 
+schedule for the activation of ZSAs on Mainnet). 
+
+The activation of ZSAs on Mainnet is also dependent on the merging of necessary 
+code changes into the core Zcash cryptographic libraries and the new Zcash stack 
+(Zebra, Zaino and Zallet) that will replace zcashd. With limited resources, 
+anything that distracts the engineers who are working on the new stack is likely 
+to delay ZSA activation further. 
+
+In addition, any changes to the core Zcash cryptographic libraries or Zebra -- 
+including the definition and implementation of a Lockbox distribution mechanism 
+-- would require that QEDIT rebase their code changes, incurring further cost 
+and delay. 
+
+This ZIP therefore proposes expressly prohibiting any disbursement of funds from 
+the Lockbox until after Zcash Shielded Assets is active on Mainnet. 
```

Comment:
It would be good that the ZIP explicitly specifies ZIPs, PRs, GitHub issues and other documented pieces of code that have to be implemented to it is possible to determine when the requirement "Zcash Shielded Assets is active on Mainnet" is completed without any ambiguity.